packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -535,11 +545,17 @@ test('price drop', async t => {
     AmountMath.make(aethBrand, 400n),
     'vault holds 11 Collateral',
   );
-  await manualTimer.tick();
+  trace(t, 'pa2', priceAuthority);
+
+  priceAuthority.setPrice(makeRatio(677n, runBrand, 900n, aethBrand));
+  trace(t, 'price dropped a little');
+  // await manualTimer.tick();
```

Comment:
any reason to leave these `tick()`s here long term?

Reply:
nope. I'll clean that up in a future PR.